declarations_site/cms_pages/templates/cms_pages/home.jinja

```diff
@@ -140,3 +76,89 @@
     <div class="cp"></div>
     <!-- clear padding --> 
 {% endblock %}
+
+{% block graph %}
+    <section id="graph" class="on-white">
+        <div class="container">
+            <div style="float: left; width: 30%; margin-top: 30px; font-size: 1.2em">ТОП-15 областей за середнім рівнем доходів чиновників. Клікніть на заголовок колонки, щоб відсортувати таблицю. Для того, щоб перейти на <a href="/BI">сторінку аналітики</a> для конкретної області, клікніть на строку в таблиці</div>
+            <link href='/static/css/bi/widget_home.css' rel='stylesheet' type='text/css'>
+            <table class="first">
+              <thead>
+                <tr class="header">
+                  <th>Регіон</th>
+                  <td style="padding-left: 80px;">Дохід</td>
```

Comment:
Please, do not use inline JS or CSS (especially if you have dedicated CSS file for that)